src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -891,7 +891,13 @@ public static RunConfiguration FromXml(XmlReader reader)
 
                     case "DotNetHostPath":
```

Comment:
I think there's a bigger problem at hand here. This uses upper-case `N`, which means that the XML is only valid when it has upper-case N in the setting.

But it sets lower-case N here.